light-prover/integration_test.go

```diff
@@ -372,3 +375,72 @@ func TestCombinedHappyPath_JSON(t *testing.T) {
 		t.Fatalf("Expected status code %d, got %d %s", http.StatusOK, response.StatusCode, string(responseBody))
 	}
 }
+
+func TestBatchAppendHappyPath(t *testing.T) {
+	treeDepth := uint32(26)
+	batchSize := uint32(1000)
+	startIndex := uint32(0)
+	params := prover.BuildAndUpdateBatchAppendParameters(treeDepth, batchSize, startIndex, nil)
+
+	jsonBytes, _ := params.MarshalJSON()
+
+	response, err := http.Post(proveEndpoint(), "application/json", bytes.NewBuffer(jsonBytes))
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer response.Body.Close()
+
+	if response.StatusCode != http.StatusOK {
+		body, _ := io.ReadAll(response.Body)
+		t.Fatalf("Expected status code %d, got %d. Response body: %s", http.StatusOK, response.StatusCode, string(body))
+	}
+}
+
+func TestBatchAppendLargeInput(t *testing.T) {
```

Comment:
what's the difference between `TestBatchAppendHappyPath` and `TestBatchAppendLargeInput`?

Reply:
Leftover, removed, thanks.